### ISSUE TITLE
Create an API for Bukkit plugins

### ISSUE THREAD

**Issue by lokka30** [ArcanePlugins/BlackWidow#6]
It may be beneficial to offer an API so Bukkit plugins can integrate with BlackWidow's bukkit plugin implementation. This may require a re-write of portions of the Bukkit plugin code-base.

This should be completed before #2. I believe this is still lower priority than #1, even if portions of that code need to be rewritten.
